### ISSUE TITLE
faker.fake : {{random.number(x)}} fails

### ISSUE THREAD

**Issue by PaulVanStaden** [json-schema-faker/json-schema-faker#259]
1. The "random" functionality disappeared in version 4.1.0

http://json-schema-faker.js.org/#gist/0d442502ae42a5877c65e4e6ee1e6caf
{
  "type": "string",
  "faker": {
    "fake": "{{random.number(100)}}, {{name.firstName}} {{name.suffix}}"
  }
}

2. the result you get
Throws an error saying "Invalid method: random.number(100) in /"

3. the result you expected
"53, Gideon DVM"

**Comment by pateketrueke** [comment]
Fortunately we rely on faker.js for that keywords, but I guess we shipped a broken version of it on `4.1.0` so the only solution is upgrading manually to a stable version of faker.js, and if necessary, registering the dependency.

On versions from `0.5.x` and beyond those dependencies are not longer installed with `jsf`, so you need to do this for loading keyword extensions.

BTW; I just tested it on the latest branch and works perfectly (`json-schema-faker@0.5.0-rc1` + `faker@4.1.0`)

```js
const schema = {
  "type": "string",
  "faker": {
    "fake": "{{random.number(100)}}, {{name.firstName}} {{name.suffix}}"
  }
};

const jsf = require('json-schema-faker');

jsf.extend('faker', () => require('faker'));

jsf(schema).then(result => console.log(JSON.stringify(result, null, 2)))
```

**Comment by PaulVanStaden** [comment]
Hi, 

thanks for the reply. I will update to 0.5.x shortly and see if my problem is resolved
